alpaca/trading/enums.py

```diff
@@ -317,9 +317,11 @@ class TradeConfirmationEmail(str, Enum):
     """
     Used for controlling when an Account will receive a trade confirmation email.
 
-    please see https://alpaca.markets/docs/api-references/broker-api/trading/trading-configurations/#attributes
+    please see https://docs.alpaca.markets/reference/getaccountconfig
     for more info.
     """
 
     ALL = "all"
     NONE = "none"
+    # please do not use this value, to support legacy code
+    ENABLED = "enabled"  # should not be used
```

Comment:
Normally a new line at the end of the file.

Reply:
reverted!